c/misra/test/rules/DIR-4-9/test.c

```diff
@@ -0,0 +1,30 @@
+#define MACRO(OP, L, R) ((L)OP(R))   // COMPLIANT
```

Comment:
Overall, I think we should add more non-compliant cases.

Reply:
sure, I added one other case, and actually one more compliant (when there is no body, I think that wont make sense to replace).

I can't think of any other edge cases to represent though, so if you had any suggestions ?

Reply:
Maybe some FNs like `#define MY_ASSERT(X) assert(x)` to show what we don't handle currently?

Reply:
added that one! good point, that is a case where it should be just the function call.

Reply:
Was a completely reasonable ask because my comment should have provided more information 😅.